SU-HSDO/suhumsci
# Four Kitchens Release - Sprint 44

# READY FOR REVIEW

## Summary
- SHS-5052 - Video preview in CKeditor should better reflect rendered size
- SHS-5491 - Implementation: Menu UI updates
- SHS-5495 - Update border color on combo based select fields
- SHS-5492 - Bug: hb-stretch-vertical-linked-cards class not resizing cards properly
- SHS-4898 - [EX] Add link to user guide on all content types
- SHS-5053 - Consistency: Update to field names

## Need Review By (Date)
02/28

## Urgency
medium

## Steps to Test

### SHS-5052

1. Go to any page, edit it or create a new one
2. In a WYSIWYG area, embed a video and select the "Align center and break text" alignment. 
3. The preview of the video should be the full width of the CKEditor area

### SHS-5491

1. Create a new Flexible page.
2. Check the "Provide a menu link" field in the right sidebar
3. Confirm that each the menu hierarchy selector levels have a label and a help text. Confirm that the texts match the specs in the [design ticket](https://app.clickup.com/t/36718269/SHS-5489)
4. Confirm that the style of the menu hierarchy selector elements match the requirements (see screenshot)
5. Repeat the previous steps for the edit form of an existing Flexible page

<img width="331" alt="Screenshot 2024-02-19 at 14 58 51" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/251341c9-1241-442c-a8f8-2299c850b402">

### SHS-5495

1. Go to the following pages:
   - https://feminist.suhumsci.loc/news-events/upcoming-events
   - https://creativewriting.suhumsci.loc/books
   - https://hs-colorful.suhumsci.loc/default-views/courses
   - https://hs-traditional.suhumsci.loc/default-views/courses
2. Confirm the border color of the select fields match the border color of the Search by Keyword textfield

### SHS-5492

1. Go to https://english.suhumsci.loc/
2. Log in and go to https://english.suhumsci.loc/node/2/layout
3. Remove the `.hb-raised-cards--uniform-height` class in the `Vertical Linked Cards`
4. Confirm the cards and their titles have the same height

### SHS-4898

1. Verify that the `User Guide` field group is open and the `User Guide` link opens a new tab/window to `https://hsweb.slite.page/p/5Qd9ZB~jGu/H-and-S-Drupal-Platform-User-Guide` for all content types in the following scenarios
2. Creating new content
3. Editing existing content
4. Using different themes, i.e. colorful, traditional

### SHS-5053

1. Changes `Spotlight` component label and add more label from `Slides` to `Spotlight(s)`
2. Changes `Gradient Hero Link` label to `Link` in the `Banner image with overlay and text` components
## Steps to Test
1. Create a new Flexible Page
2. Add a `Spotlight` component to the `Main Region`
3. Verify the widget label and add more label show `Spotlight(s)` instead of `Slides`
4. Add a ` Banner image(s) with partial overlay and text` component to both the `Main Region` and the `Full-width Region`
5. Add multiple `Banner image with overlay and text` components to each `Banner image(s) with partial overlay and text` component
6. Verify that each instance of `Banner image with overlay and text` link label shows `Link` instead of `Gradient Hero Link`

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
